bundle/config/root_test.go

```diff
@@ -163,3 +167,60 @@ func TestRootMergeEnvironmentWithMode(t *testing.T) {
 	require.NoError(t, root.MergeEnvironment(env))
 	assert.Equal(t, Development, root.Bundle.Mode)
 }
+
+func TestConfigFileNames_FindInPath(t *testing.T) {
+	testCases := []struct {
+		name     string
+		files    []string
+		expected string
+		err      string
+	}{
+		{
+			name:     "file found",
+			files:    []string{"databricks.yaml"},
+			expected: "BASE/databricks.yaml",
+			err:      "",
+		},
```

Comment:
Could you add a testcase for `bundle.yml` as well? That's what folks use today so good to double check.

Reply:
yes